src/plugins/api-proxy/util/receive.js

```diff
@@ -122,9 +122,13 @@ export const parseApiValue = ([response, body]) => {
 
 	try {
 		const value = parseBody(body);
-		if (response.statusCode < 200 || response.statusCode > 399) {
+		if (response.statusCode < 200 || response.statusCode >= 500) {
 			return formatApiError(new Error(response.statusMessage), value);
 		}
+		if (response.statusCode >= 400) {
+			// Don't throw an error for 400s for now (maybe forever?)
+			return formatApiError({ message: response.statusMessage }, value);
+		}
```

Comment:
this has the same effect of the previous implementation - `formatApiError` expects an `Error` instance and will take the `message` prop from it without `throw`ing.

Reply:
Yeah, I did this first and it I don't think it's helpful. Removed.